src/components/recruting/_02_prepare/_05/CreateApplicationFormContainer.tsx

```
@@ -1,109 +1,126 @@
+        caution: string;
+        questions: Record<string, Question>;
+      }
+    >
+  >(
+    group.reduce(
+      (acc, g) => ({
+        ...acc,
+        [g.name]: {
+          caution: "",
+          questions: {
+            [uuidv4()]: { ...initialQuestion }
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)